### ISSUE TITLE
Implicit query type cant be set to Number

### ISSUE THREAD

**Issue by iveselin** [nestjs/swagger#189]
## I'm submitting a...

<pre><code>
[ ] Regression 
[ x] Bug report
[ ] Feature request
[ ] Documentation issue or request
</code></pre>

## Current behavior

When setting \@ApiImplicitQuery with array set to true, type cant be set to Number, but has to be wrapped in qoutes in which case type validation in docs doesnt work

## Expected behavior

Same behavior like in ApiModelProperty where you can set type to Number and then input in docs is validated

## Minimal reproduction of the problem with instructions

Create a simple route which accepts an array for query (ex. array of ids) and set type to Number
```
@ApiImplicitQuery({ name: 'ids[]', isArray: true, type: Number, collectionFormat: "multi", required: false })
    async index(
        @Query() ids:number[]
    ) {
        //#index get route
        return await this.projectsService.findAll(ids);
    }
```
## What is the motivation / use case for changing the behavior?

Bug in docs

## Environment

<pre><code>
Nest version: 5.4.0

 
For Tooling issues:
- Node version: 10.15.0  
- Platform:  Windows

Others:

</code></pre>

**Comment by kamilmysliwiec** [comment]
Please, provide a minimal repository which reproduces your issue.

**Comment by iveselin** [comment]
Reproduction can be found [here](https://github.com/iveselin/nestswaggerbug/tree/development) on development branch, dogs swagger wont show, cats is working and type number is validated.

**Comment by kamilmysliwiec** [comment]
Fixed in 4.0.0
